<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KayitWidget</class>
 <widget class="QWidget" name="KayitWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1020</width>
    <height>620</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Kayıt Ol</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_main">
   <item>
    <spacer name="verticalSpacer_top">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item alignment="Qt::AlignmentFlag::AlignHCenter">
    <widget class="QFrame" name="kayit_form_frame">
     <property name="minimumSize">
      <size>
       <width>400</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>400</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #363a45; border-radius: 8px; padding: 20px;</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_form">
      <property name="spacing">
       <number>15</number>
      </property>
      <item>
       <widget class="QLabel" name="baslik_label">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:16pt; font-weight:600; color:white;&quot;&gt;Hesap Oluştur&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="kullanici_adi_input">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>35</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: #495057; border: 1px solid #5c6370; border-radius: 5px; padding: 5px; color: white;</string>
        </property>
        <property name="placeholderText">
         <string>Kullanıcı Adı</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="email_input">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>35</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: #495057; border: 1px solid #5c6370; border-radius: 5px; padding: 5px; color: white;</string>
        </property>
        <property name="placeholderText">
         <string>E-posta Adresi</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="sifre_input">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>35</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: #495057; border: 1px solid #5c6370; border-radius: 5px; padding: 5px; color: white;</string>
        </property>
        <property name="echoMode">
         <enum>QLineEdit::EchoMode::Password</enum>
        </property>
        <property name="placeholderText">
         <string>Şifre</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="sifre_tekrar_input">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>35</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: #495057; border: 1px solid #5c6370; border-radius: 5px; padding: 5px; color: white;</string>
        </property>
        <property name="echoMode">
         <enum>QLineEdit::EchoMode::Password</enum>
        </property>
        <property name="placeholderText">
         <string>Şifre (Tekrar)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="kayit_tamamla_buton">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>40</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
        background-color: #28a745;
        border-radius: 5px;
        color: white;
        font-weight: bold;
        font-size: 14px;
}
QPushButton:hover {
        background-color: #218838;
}
QPushButton:pressed {
        background-color: #1e7e34;
}</string>
        </property>
        <property name="text">
         <string>Kayıt Ol</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="girise_don_buton">
        <property name="styleSheet">
         <string notr="true">QPushButton {
        background-color: transparent;
        color: #3498db;
        text-align: center;
}
QPushButton:hover {
        color: #5dade2;
        text-decoration: underline;
}</string>
        </property>
        <property name="text">
         <string>Zaten bir hesabın var mı? Giriş Yap</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_bottom">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
